bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/observability/ObservabilityConstants.java

```diff
@@ -28,6 +28,7 @@ public class ObservabilityConstants {
 
     private ObservabilityConstants() {
     }
+    public static final String BAL_OBSERVE_SERVICE_NAME_SUFFIX = "BAL_OBSERVE_SERVICE_NAME_SUFFIX";
```

Comment:
We don't need to make it public right? Package private would be enough IMO :)

Reply:
@sameerajayasoma The `ObservabilityConstants` class is private. Therefore we can access the Constant only within package AFAIK ?

Reply:
@sameerajayasoma Yes. We don't need it to make public. Addressed the suggestions